cg/services/order_validation_service/response_mapper.py

```diff
@@ -74,8 +93,8 @@ def wrap_case_fields(case: dict) -> None:
             set_field(entity=case, field=field, value=value)
 
 
-def wrap_sample_fields(case: dict) -> None:
-    samples: list[dict] = case.get("samples", [])
+def wrap_sample_fields(dict_containing_samples: dict) -> None:
+    samples: list[dict] = dict_containing_samples.get("samples", [])
```

Comment:
This got quite ugly but essentially the only input needed is a dict with a key called "samples". Any way to make it better?

Reply:
Does the input need to be a dict? Can we not just pass the samples value if it exists? Or was there a decision to only operate on the whole order?

Reply:
ooo that sounds promising! I will give it a shot

Reply:
Change worked!